.github/workflows/changelog.yml

```diff
@@ -0,0 +1,51 @@
+name: Changelog
```

Comment:
Out of curiosity, why did you choose to make this a changelog workflow rather than a release workflow (by naming that is)?

Reply:
I was under the impression that `name` was simply what was displayed in the GitHub UI and I thought that a `Changelog` was more descriptive. If there are other implications for this field, I can adjust this.

Reply:
Oops sorry I should have been more specific, I was really commenting on the file as a whole (I wish there was a better way to do that in GitHub). I wasn't talking necessarily about this particular field, but rather this workflow in general. It looks like the open source site has this under their `release` workflow alongside some of the other release related items like generating the third party notices. 

I was mostly just curious why this was its own workflow instead of part of a release workflow.